keyboards/lyra/rev1/rev1.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2021 Domanic Calleja
```

Comment:
`rev1.c` is empty. It needs to have at least this header and `#include "rev1.h"`.

Reply:
Drashna removed it in their change request but I have put it back
https://github.com/qmk/qmk_firmware/pull/13977/commits/b1d1325767f7a84f33b9066607822df39bd18393